Userland/Libraries/LibJS/Bytecode/BasicBlock.h

```diff
@@ -36,18 +36,19 @@ class BasicBlock {
     bool can_grow(size_t additional_size) const { return m_buffer_size + additional_size <= m_buffer_capacity; }
```

Comment:
Typos in commit message: 
`LibJS: Remeber which which instruction terminated a block`